server/services/caseloadService.ts

```diff
@@ -49,7 +52,10 @@ export default class CaseloadService {
       .filter(offender => offender.status && offender.status.startsWith('ACTIVE'))
       .filter(offender => !offender.indeterminateSentence && offender.conditionalReleaseDate)
       .filter(offender => !offender.releaseDate || moment().isBefore(moment(offender.releaseDate, 'YYYY-MM-DD')))
-      .filter(offender => !offender.homeDetentionCurfewEndDate)
+      .filter(offender => {
+        const hdcStatus = hdcStatuses.find(hdc => hdc.bookingId === offender.bookingId)
+        return hdcStatus?.eligibleForHdc ? !hdcStatus.eligibleForHdc : true
```

Comment:
Is this more convoluted than it needs to be? is this enough?

```
return !hdcStatus?.eligibleForHdc
```

Reply:
Yes, probably! I'll alter and verify with the tests :-)